DashDocs/src/utils/macro.jl

```diff
@@ -0,0 +1,52 @@
+macro eval_toplevel()
+    result = Expr(:block)
+    result.args = TOP_LEVEL
+    return esc(result)
+end
+
+function source_formatted(source)
+
+end
+
+macro source_str(name)
+    source_str = "```julia\n$(EXAMPLES_REGISTRY[name].source)\n```"
+    return :(
+        html_div(
+            dcc_markdown($source_str),
+            className = "code-container",
+            style = Dict("borderLeft" => "thin lightgrey solid"),
+        )
+    )
+end
+macro layout_str(name)
+    return :(
+            EXAMPLES_REGISTRY[$name].layout()
+    )
+end
+
+macro pkgver_str(name)
```

Comment:
This will include all the packages in the Manifest which are not direct dependencies. Is this an issue?

Reply:
To be honest, I didn't really understand what the issue was in this section. This question is more likely to relate to `eval_toplevel`. Yes, there is an expectation that all the packages that are found in the examples should be in the DashDocs dependencies

Reply:
But should they be in the direct dependencies? This method will not complain if the package is in the Manifest but you still can't call `using PackageName` because it's not a direct dependency.

Reply:
Yes, they should be in direct dependencies. I consider the examples to be part of the package and it seems logical to me that what is used inside the package should be in the dependencies. In principle, I can add a little more magic and do an automatic `Pkg.add` for such things, but I don't think it makes sense to waste time on this now.

Reply:
Ok I will merge but please open an issue to track this.